package.json

```diff
@@ -511,6 +655,31 @@
             "type": "string"
           }
         },
+        "r.rMarkdownCodeLens.option": {
```

Comment:
Does it make sense that the settings for RMarkdown is named under "r.rmarkdown.*" as `rmarkdown` is often considered a term in this case?

Reply:
Also, this particular setting is about the chunk commands provided by CodeLens, I suggest that it is named `r.rmarkdown.codeLensCommands`? Or alternatively, `r.rmarkdown.chunkCommands` as many users might not know what is CodeLens?

Reply:
Yea totally agree. Renamed to `r.rmarkdown.codeLensOptions`

Reply:
This isn't resolved yet. I think `r.rmarkdown.codeLensCommands` sounds better than `r.rmarkdown.chunkCommands` because `chunkCommands` is more generic, but the settings is specific to CodeLens. What do you think?

Reply:
`r.rmarkdown.enableCodeLens` also refers to CodeLens, so `r.rmarkdown.codeLensCommands` sounds more consistent with it.

Reply:
I added some description for this

![99](https://user-images.githubusercontent.com/54139483/100493420-c37e1500-318a-11eb-8553-86d9cae71fa6.png)

Reply:
Yes, I agree. `r.rmarkdown.enableCodeLens` and `r.rmarkdown.codeLensCommands` sound more consistent.